SocialCareCaseViewerApi.Tests/V1/Boundary/Request/UpdateMashReferralTests.cs

```diff
@@ -85,7 +85,7 @@ public void WhenUpdateTypeIsInitialDecisionRequiresUrgentContactMustBeProvided()
         [Test]
         public void WhenUpdateTypeIsInitialDecisionReferralCategoryMustNotBeNull()
```

Comment:
This test reads initial decision in the description but we pass "FINAL-DECISION" as the update type within the test.